plugin/src/main/scala/com/lightbend/sbt/javaformatter/JavaFormatter.scala

```diff
@@ -31,9 +30,13 @@ object JavaFormatter {
       includeFilter: FileFilter,
       excludeFilter: FileFilter,
       streams: TaskStreams,
-      cacheStoreFactory: CacheStoreFactory): Unit = {
+      cacheStoreFactory: CacheStoreFactory,
+      aospStyle: Boolean): Unit = {
     val files = sourceDirectories.descendantsExcept(includeFilter, excludeFilter).get.toList
-    cachedFormatSources(cacheStoreFactory, files, streams.log)(new Formatter)
+    val options =
```

Comment:
IMHO, this should be a separate setting in the plugin, and the functions here should accept *it* as a parameter, rather than the `aospStyle` flag.